### ISSUE TITLE
Docs unclear on how to correctly `import` an existing Observable notebook

### ISSUE THREAD

**Issue by shafdog** [observablehq/framework#1128]
In reading the docs, "How to use an Observable Notebook in Framework", is not covered under [JavaScript: Imports](https://observablehq.com/framework/javascript/imports#javascript%3A-imports), nor elsewhere.

In fairness, "cut-and-paste" the Notebook's Embed into a Framework code block works. But I could not figure some direct `import` to include the top-level reactive variables into Framework's `.md` files, without writing JS & `Runtime().module()` to load them "manually".  While one Notebook can import another Notebook, and use the imported notebook's variables in one-line, how to extend that to Framework was unclear. 

To me, being able to use Notebook for the more logic/code intensive stuff, and Framework for text/presentation/"packaging" stuff... seems like a natural fit. 
_i.e._ where a Notebook is a REPL, with Framework being semi-static runtime that use the Notebook's "code".   

Perhaps this is more ObservableHQ feature request to deal with Framework better.  Or I'm missing something is totally possible too.

**Comment by mbostock** [comment]
Importing Observable notebooks is not supported.

You can use the `observable convert` command to assist in downloading an Observable notebook and converting it to Framework-compatible Markdown. Because Observable notebooks use nonstandard Observable JavaScript, you may have to change the code to make it compatible with Framework’s vanilla JavaScript.

Forum thread here: https://talk.observablehq.com/t/d3-swatches-imports-in-framework/8893

**Comment by shafdog** [comment]
Fair enough.  I kinda forgot that:
> Observable notebooks use nonstandard Observable JavaScript

More experimenting, than need specific here.  Really like Framework's Data loader concept — imagine that be another way to bring in [any] notebook actually.

**Comment by mbostock** [comment]
Thanks for your understanding.

We are interested in making it easier to develop and share components across projects. But we’re favoring more standard (vanilla) components in Framework. You can publish components to npm and then import them, for example.

And eventually we’d like to bring Framework’s vanilla syntax back to notebooks, and thereby make the transition between Framework and notebooks more fluid, but that’ll be some work (versioning) so it’ll be a while before that happens.

**Comment by wiless** [comment]
I had recently switched to "svelte" to leverage the "reactiveness" and auto-execution of js-cells. I had to write some wrap-around functions to accomodate the  Observable notebook's features like  `viewof` for inputs and custom "display" function to show variables.  

I love the new observablehq framework that natively supports these features using `view` and `display` functions. The ability to import the entire notebook as module or even reuse the cell from an online observablehq notebook  would be great.

For e.g.
```
 import {{HexGrid,hexPlot as hp,Hex}} from "https://api.observablehq.com/@wiless/drawutils.js?v=4"
```

or 

```
 import {{HexGrid,hexPlot as hp,Hex}} from "obs:@wiless/drawutils" 
```
